.github/workflows/screener-build.yml

```diff
@@ -75,19 +84,34 @@ jobs:
 
   screener-react-northstar:
     runs-on: 'ubuntu-latest'
-
     name: Screener @fluentui/react-northstar
-
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - name: Log environment variables (Linux)
-        if: runner.os == 'Linux'
+      - name: Set env variables if there is a PR
         run: |
-          printenv | sort ;\
-          echo "SHELLOPTS $SHELLOPTS" ;\
+          echo "SYSTEM_PULLREQUEST_TARGETBRANCH=${{ github.event.pull_request.base.ref }}" >> $GITHUB_ENV
+          echo "SYSTEM_PULLREQUEST_SOURCEBRANCH=${{ github.event.pull_request.head.ref }}" >> $GITHUB_ENV
+          echo "SYSTEM_PULLREQUEST_PULLREQUESTID=${{ github.event.pull_request.id }}" >> $GITHUB_ENV
+          echo "SYSTEM_PULLREQUEST_SOURCECOMMITID=${{ github.event.pull_request.head.sha }}" >> $GITHUB_ENV
+          echo "ISPR=true" >> $GITHUB_ENV
```

Comment:
As I see `ISPR` is used only inside this file, can we remove it and replace its usages with `if: ${{!startsWith(github.ref, 'refs/pull/')}}`?

Reply:
sure, I will replace that